### PR TITLE
Vendors: List potential vendors and convert Organization to Vendor

### PR DESCRIPTION
**Pull request by kewitz**
Requires https://github.com/opencollective/opencollective-api/pull/9486

### CODE REVIEW

File: components/Avatar.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ const Avatar = ({
       src = defaultImage.ANONYMOUS;
     } else if (collective.isGuest && shouldUseDefaultGuestAvatar(collective.name)) {
       src = defaultImage.GUEST;
-    } else if (type === 'VENDOR') {
+    } else if (type === 'VENDOR' && !collective.imageUrl) {
```
Comment (Betree) [inline components/Avatar.tsx:85]: Isn't `imageUrl` always set? I don't think this field returns `null` if the account has no uploaded image https://github.com/opencollective/opencollective-api/blob/debt/remove-create-pledge/server/graphql/v2/interface/Account.ts#L802

File: components/dashboard/sections/Vendors.tsx [hunk 2]
```diff
@@ -79,24 +87,56 @@ const dashboardVendorsQuery = gql`
               ...VendorFields
             }
           }
+          potentialVendors @include(if: $includePotentialVendors) {
```
Comment (Betree) [inline components/dashboard/sections/Vendors.tsx:90]: I love these directives

**Review by Betree — COMMENTED** [review]
Early review, will look at the API then test the whole

File: components/Avatar.tsx (hunk 1 above)
Comment (kewitz) [reply components/Avatar.tsx:85]: We're not really allowing users to customize the image for newly created vendors, in this context, what happens if the `collective.image` is empty? Well, what role does the `collective.image` even plays?

Comment (Betree) [reply components/Avatar.tsx:85]: `collective.imageUrl` resolver uses `getImageUrl` which calls `getCollectiveAvatarUrl` https://github.com/opencollective/opencollective-api/blob/debt/remove-create-pledge/server/lib/collectivelib.ts#L38. This function always returns a string, never null, even if there's no image. `collective.image` is included as the hash in the URL, if available.

**Review by Betree — APPROVED** [review]
Good stuff!

**Pull request merged**